ppr-ui/src/views/amendment/AmendRegistration.vue

```diff
@@ -503,6 +512,12 @@ export default defineComponent({
 
     const hasAmendmentChanged = (): boolean => {
       let hasChanged = false
+
+      if (isSecurityActNotice.value) {
+        if (!isEqual(getSecuritiesActNotices.value, getOriginalSecuritiesActNotices.value)) {
```

Comment:
These two conditions could be combined into one.

Reply:
Definitely, will do, thank you!